src/monstergenerator.h

```diff
@@ -28,13 +28,13 @@ struct species_type {
     species_id id;
     bool was_loaded = false;
     translation description;
-    std::string footsteps;
+    translation footsteps;
     enum_bitset<m_flag> flags;
     enum_bitset<mon_trigger> anger;
     enum_bitset<mon_trigger> fear;
     enum_bitset<mon_trigger> placate;
     std::string get_footsteps() const {
-        return footsteps;
+        return footsteps.translated();
```

Comment:
I suggested translating the string in this function in https://github.com/CleverRaven/Cataclysm-DDA/pull/31493#discussion_r294059886, but that was rejected. What has changed?

Reply:
I can profile performance again, but I don't think anything was changed since that time. My opinion is that we still need to translate this string only once during loading from json (unless we do something with performance).

Reply:
I don’t know what to do now, so I’ll take off WIP status. I just wanted to add some translation tags first. But after the code review we got this. I did not notice any drawdowns in performance, but I have an i7 processor.

Reply:
I decided to address possible performance issues later and merge it as is.